dvc/data/_progress.py

```diff
@@ -0,0 +1,35 @@
+from dvc.progress import Tqdm
+
+
+class QueryingProgress(Tqdm):
+    def __init__(self, *args, name=None, **kwargs):
+        msg_part = "cache in " + f"'{name}'" if name else "remote cache"
+        self._estimating_msg = f"Estimating size of {msg_part}"
+        self._listing_msg = f"Querying {msg_part}"
+        # msg to show something before the callback gets called
+        kwargs.setdefault("desc", f"Checking {msg_part}")
+        super().__init__(*args, **kwargs)
+
+    def _progress(self, msg, *args) -> None:
+        total = args[0] if args else self.total
+        completed = args[1] if len(args) > 1 else self.n
```

Comment:
The progress bars take `completed`/`current`/`absolute` value rather than a relative/`to_increment` value. Maybe I am overengineering here, but in the future, we may want to throttle callbacks, so I felt like we should use `absolute` values in most places, instead of keeping callbacks guessing.